apps/explorer/utils/__init__.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas
+import yaml
+import zipfile
+
+from io import BytesIO, StringIO
+
+
+def export_pixelsets(pixel_sets):
+    pixelsets_meta = {}
+
+    omics_unit_col = 'Omics Unit'
+    df = pandas.DataFrame(columns=(omics_unit_col,))
+
+    for index, pixel_set in enumerate(pixel_sets):
+        pixel_set_id = pixel_set.get_short_uuid()
+
+        # add metadata for this pixel set
+        pixelsets_meta[pixel_set_id] = {
+            'pixelset': pixel_set_id,
+            'description': pixel_set.description,
+            'columns': [index * 2 + 1, index * 2 + 2],
```

Comment:
Even if multiplication has a higher precedence than addition, I would prefer to be more explicit: 

`[(2 * index) + 1, (2 * index) + 2]`